src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java

```diff
@@ -30,35 +31,77 @@ public void sendMenteeApplicationEmails(long id, Optional<Program> program) thro
+                        "again next year and follow us on our social media channels for future programs. " +
+                        "If you have any clarifications, please reach out to us via " +
+                        "<a href=\"mailto:sustainableedufoundation@gmail.com\">sustainableedufoundation@gmail.com</a>";
+
+                emailService.sendEmail(mentor.getProfile().getEmail(), StringUtils.capitalize(mentor.getState().name()), message, false);
+
+            }
         }
     }
 
     public void sendMenteeSelectionEmails(long id, Optional<Program> program) throws IOException, MessagingException {
         List<Mentor> approvedMentors = mentorRepository.findAllByProgramIdAndState(id, EnrolmentState.APPROVED);
 
-        String message = "You can approve or reject your mentees by visiting the dashboard";
         for (Mentor mentor : approvedMentors) {
-            emailService.sendEmail(mentor.getProfile().getEmail(), program.get().getTitle(), message);
+
+            String message = "Dear " + mentor.getProfile().getFirstName() + ",<br /><br />" +
+                    "You have student applications waiting to be reviewed. You can approve or reject your mentees " +
+                    "by visiting the <b>ScholarX dashboard.</b>";
+
+            emailService.sendEmail(mentor.getProfile().getEmail(), program.get().getTitle(), message, true);
         }
     }
 
     public void sendOnGoingEmails(long id, Optional<Program> program) throws IOException, MessagingException {
         List<Mentor> approvedMentors = mentorRepository.findAllByProgramIdAndState(id, EnrolmentState.APPROVED);
 
-        String message = "You can check your mentees by visiting the dashboard";
         for (Mentor mentor : approvedMentors) {
-            emailService.sendEmail(mentor.getProfile().getEmail(), program.get().getTitle(), message);
+
+            String message = "Dear " + mentor.getProfile().getFirstName() + ",<br /><br />" +
+                    "<b>Congratulations!</b><br />Students have accepted you as their mentor. " +
+                    "You can check your mentees and their contact details by visiting the <b>ScholarX dashboard.</b> " +
+                    "Please make the first contact with them as we have instructed them to wait for your email.";
+
+            emailService.sendEmail(mentor.getProfile().getEmail(), program.get().getTitle(), message, true);
         }
     }
 
     public void sendMentorConfirmationEmails(long id, Optional<Program> program) throws IOException, MessagingException {
         List<Mentee> mentees = menteeRepository.findAllByProgramId(id);
 
-        String message = "You can check your mentor by visiting the dashboard";
+
         for (Mentee mentee : mentees) {
-            emailService.sendEmail(mentee.getProfile().getEmail(), program.get().getTitle(), message);
+            String message = "Dear " + mentee.getProfile().getFirstName() + ",<br /><br />" +
```

Comment:
I think this one is in the wrong state. When the program is moved to the `MENTOR_CONFIRMATION` state, the mentees with multiple mentors can confirm one mentor. Mentees with only one mentor don't have to do anything.
(An email with this content can be sent to all mentees when the program state changes to `MENTEE_SELECTION`)

cc: @anjula-sack @Gravewalker666

Reply:
Yeah, true but even earlier we had a similar message right?

Reply:
It does make sense to send this email in the MENTEE_SELECTION right? I think it's better if we send a different email in the mentor confirmation. 
@akshika47 what do you think??

Reply:
I checked the google doc again, this should be sent in the MENTEE_SELECTION period. @cmdrGuyson could you please make that change?

Reply:
I don't think there's an email body written in the doc for this exact scenario at the moment.

Reply:
Earlier it was "You can check your mentor by visiting the dashboard". So the users have to check the dashboard and choose one mentor if they have multiple approvals. But now it says that the mentees are still under review.

Reply:
Yup, there's one missing email missing and an additional one too in the google doc.

Reply:
@Gravewalker666 @Piumal1999 I'll send this to mentees in the MENTEE_SELECTION period. What should I change this content to though?

Reply:
Nope don't change the content just change the state as you mentioned. :)

Reply:
I mean the present state. If this content gets moved to MENTEE_SELECTION. What should be said here?

Reply:
Oh yeah, @akshika47 can you update the google doc with a message to be sent to the mentees after mentors select their mentees??